### PR TITLE
 Partners Section 

### PR DESCRIPTION
**Pull request by Bushra369**
Create an "OurPartnersSection" component for displaying partner logos and ensure comprehensive testing.

desktop view:
image

mobile view:
image

### CODE REVIEW

**Review by riadhmouamnia — APPROVED** [review]
## Review for "Partners Section" PR

**Reviewer:** @riadhmouamnia 

Great job on the "Partners Section"! The layout of the partner images is well-executed, and everything looks visually appealing. I'm pleased to see that all partner images are in place and that all tests have passed successfully.

Your attention to detail and dedication to quality is evident in this section.

Wait for one more review from a teammate and then go for merging 🚀. Well done!

@riadhmouamnia

**Review by rahemSorour — APPROVED** [review]
Every thing looks perfect

**Pull request merged**
